code/api/src/main/scala/com/avast/datadog4s/api/event/package.scala

```diff
@@ -0,0 +1,27 @@
+package com.avast.datadog4s.api
+
+import enumeratum.{ Enum, EnumEntry }
+
+package object event {
+  sealed trait Priority extends EnumEntry
+
+  object Priority extends Enum[Priority] {
```

Comment:
I think we should have a good use case to introduce a dependency and i don't see it right now. Is there a reason we need enumeratum? I think we can get away pretty easily with using just case objects and sealed traits. We will never lookup these by string or something similar.

Reply:
good point, force of habit to import enumeratum